recipes/varscan/2.3.7/varscan.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
If you are eager to port this over to the new python wrapper: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#java

This would be great.